packages/flutter_test/lib/src/matchers.dart

```diff
@@ -249,8 +249,12 @@ Matcher isMethodCall(String name, {@required dynamic arguments}) {
 Matcher coversSameAreaAs(Path expectedPath, {@required Rect areaToCompare, int sampleSize = 20})
   => new _CoversSameAreaAs(expectedPath, areaToCompare: areaToCompare, sampleSize: sampleSize);
 
-/// Asserts that a [Finder] matches exactly one widget whose rendered image
-/// matches the golden image file identified by [key].
+/// Asserts that a [Finder] or [FutureOr<ui.Image>] matches the golden image
```

Comment:
please don't mention FutureOr in the docs. Just say `[Future<ui.Image>] or [ui.Image]` or some such.

Reply:
done